web/docs/tql2/operators/http.md

```diff
@@ -0,0 +1,123 @@
+# http
+
+Sends HTTP/1.1 requests.
+
+## Synopsis
+
+```tql
+http url:string, [method=string, payload=string, headers=record, tls=bool,
+    certfile=string, keyfile=string, password=string,
+    insert_metadata_into=field, connection_timeout=duration,
+    max_retry_count=int, retry_delay=duration, parallel=int, { … }]
+```
+
+## Description
+
+The `http` operator issues HTTP/1.1 requests and forwards received responses as events.
+
+### `url: string`
+
+URL to connect to.
+
+### `method = string (optional)`
+
+One of the following HTTP method to use when using the client:
+- `get`
+- `head`
+- `post`
+- `put`
+- `del`
+- `connect`
+- `options`
+- `trace`
```

Comment:
HTTP verbs are usually spelled all upper case. Is that transparently supported?

Reply:
No